spring-batch-notion/pom.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>3.3.5</version>
+    <relativePath/>
+  </parent>
+
+  <groupId>org.springframework.batch.extensions</groupId>
+  <artifactId>spring-batch-notion</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+
+  <name>Spring Batch Notion</name>
+  <description>Spring Batch extension for Notion</description>
+  <url>https://github.com/spring-projects/spring-batch-extensions/spring-batch-notion</url>
+  <inceptionYear>2023</inceptionYear>
```

Comment:
Shouldn't this be 2024? I can update that on merge if you want

Reply:
Indeed, it was 2023 in my repo but not here 🙂 I do it now